README.md

```diff
@@ -71,6 +73,42 @@ vs_2017 "Update Enterprise to 15.8" do
 end
 ```
 
+You can add workloads or components by using modify:
+* *Modify will always execute the installer, even if the added workload(s) already exist*
+* *It is currently not possible to remove workloads or components*
+
+
+```ruby
+vs_2017 "Add .NET Framework 4.7.2 Targeting pack" do
+  action :modify
+  version: '15.9'
```

Comment:
is it possible to make the version flexible at this point

Reply:
yes, this is just an example